articles/access_controls.md

```diff
@@ -1,58 +1,57 @@
 ---
-title: "Access and Content Restrictions"
+title: "Access Controls and Data Governance: Sharing Settings and Conditions for Use"
 layout: article
-excerpt: Learn how to share and restrict content on Synapse using the sharing settings and conditions for use. 
+excerpt: Learn how to set access controls and data governance on Synapse using `Sharing settings` and `Conditions for Use`. 
 ---
 
-## Access and Content Restrictions
-Synapse content is subject to two control settings: the `Sharing setting` and `Conditions for Use.` The `Sharing setting` determines _with whom_ content is shared. `Conditions for Use` govern _how_ content can be used. 
+## 
+Synapse has two content controls: the `Sharing setting` and `Conditions for Use.` The `Sharing setting` is an access control. It determines _who_ has access to Synapse content. `Conditions for Use` are for data governance. They determine _how_ data can be used. 
 
 ## Sharing setting
-The `Sharing setting` enables you to invite others to view or collaborate on a project you have created. Projects can be **Public** visible to all or **Private** and visible to selected few. Public projects and contents are shared **broadly** with all registered Synapse users while Private projects and content are shared **sparsely**, with a limited set of designated users (e.g., a team) or not shared at all. **By default, your newly created project is private;** you are the only person that can access it or any content included in it. You can use the `Sharing setting` to enable others to view the content of your projects.
-{% include important.html content="Users are responsible for determining the appropriate Sharing Setting for any content they upload into Synapse." %}
-
-#### Create a team
-If you want to share a project with a specific group of people, you can create a team. Any Synapse user can create a team and invite other Synapse users to join their team. Only after the invited user accepts this invitation is he/she added to the team. Similarly, any Synapse user can request to be added to a team.
+You can use the `Sharing setting` to invite people to view or collaborate on a Synapse `Project`. There are two `Sharing settings`: **Public** and **Private**. Synapse content with the **Public** `Sharing setting` is visible to all Synapse users. By contrast, the **Private** `Sharing setting` limits who can see content. **By default, new `Projects` are Private.** 
+{% include important.html content="Synapse users are responsible for determining the appropriate `Sharing Setting` for any content they upload into Synapse." %}
 
```

Comment:
Is it right of us to focus so specifically on Projects.  All "entities" can have sharing settings down to the individual file.  I know this follows the old text.  

Considering that the text "  **By default, new `Projects` are Private.**  is important, I suggest highlighting this as a note above the important text.